objects/member.php

```diff
@@ -103,6 +103,11 @@ class Member
 	 * @var int
 	 */
 	var $missedPhases;
+	
+	/**
+	 * The number of already used up NMRs by this Member that got excused

```

Comment:
Based on how this is used above "SET m.excusedMissedTurns = m.excusedMissedTurns - 1" I think this will need to hold 

/**
* The number of excused turns this member has left.

Reply:
Yes, I already realized that I assumed a different definition of member's excused missed turns than you (counting up not down). I will need to adjust the initial value on game creation and probably the display on the board page, as well.